### ISSUE TITLE
Lazy loading option in Repeater

### ISSUE THREAD

**Issue by jkarelbroer** [soflyy/oxygen-bugs-and-features#1725]
To prevent the Repeater or Smart  from causing an overdose of data load in the editor and in the front-end page, I think it would be great to have lazy loading as an option.

**Comment by KittenCodes** [comment]
This would be great to use in the builder to prevent a page or template having so many AJAX requests on the initial load.

**Comment by jamescrabb** [comment]
Yes definitely if only to prevent too many request when large post data is being queries
